_posts/2023-05-31-zfp.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Video: Fast, Accurate Data Compression for Modern Supercomputing Applications"
+categories: multimedia
+---
+
+Developed at LLNL, the [zfp software library](https://github.com/LLNL/zfp) provides a comprehensive solution to both lossy and lossless data compression. zfp reduces the storage space of high-precision floating-point data without sacrificing its accuracy. Unique among data compressors, zfp is designed to be a compact number format for storing data arrays in-memory in compressed form while still supporting high-speed random access. zfp divides multidimensional arrays into small blocks that are independently compressed and decompressed on demand when an array element is accessed, without the application’s knowledge. This flexibility allows applications to work with zfp arrays as though they were regular uncompressed arrays while saving storage, time, and compute power. [Watch a new video about zfp](https://www.youtube.com/watch?v=09Jl2ggiDuY) on YouTube (6:49).
```

Comment:
I appreciate you adding the lengths. I try to do the same and always appreciate it when I see it.